### ISSUE TITLE
LORA-E5 Pin 24 (PB13/BOOT/SCK) needs to be pulled high

### ISSUE THREAD

**Issue by dkadish** [dkadish/BioAcousticIndexTool#8]
From https://wiki.seeedstudio.com/LoRa-E5_STM32WLE5JC_Module/#application-notes

```
The AT command firmware contains a bootloader for DFU and the AT application. The "PB13/SPI_SCK/BOOT" pin is used to control Wio-E5 to stay in the bootloader or jump to the AT application. When PB13 is HIGH, the module will jump to AT application after reset, with a default baud rate of 9600. When PB13 is LOW (press the "Boot" button on Wio-E5 mini/ Wio-E5 Development Kit), the module will stay in the bootloader, and keep transmitting "C" character every 1S at baud rate 115200.
```

`PB13` needs to be pulled high so that it defaults to the AT application. There should be a jumper to pull it low as well.

**Comment by dkadish** [comment]
This doesn't seem to be causing problems currently. Add a solderable jumper to correct this, if need be.

**Comment by dkadish** [comment]
Solved with 856e9c33e074f8342c6f5b2cfc0530d1effd26fc. Still needs testing

**Comment by dkadish** [comment]
Test confirmed. Tying to GND puts it into bootloader mode. Don't think we want this, but leaving the jumper there just in case!
